VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -486,6 +487,20 @@ export class VideoCallService {
         return this.apiClient.getParticipantRoomForParticipant(conferenceId, participantId, 'Judicial').toPromise();
     }
 
+    connectWowzaAgent(ingestUrl: string, callbackFn: Function) {
+        const params = {
+            streaming: true,
+            call_type: 'audio',
+            remote_display_name: this.wowzaAgentName
+        };
+        this.pexipAPI.dialOut(ingestUrl, 'auto', '', callbackFn, params);
+    }
+
+    disconnectWowzaAgent(wowzaUUID: string) {
```

Comment:
Should this be committed if it's for testing purposes? If so, maybe add method doc/summary so it's clear in case someone tries to use the method but see's the description in the intelli-sense. We can't guarantee everyone will look into the code to see the comment